### PR TITLE
[🐴] show deleted account for `missing.invalid`

### PR DESCRIPTION
**Pull request by haileyok**
Screenshot 2024-05-14 at 11 11 34 AM
Screenshot 2024-05-14 at 11 11 38 AM

### CODE REVIEW

File: src/screens/Messages/List/ChatListItem.tsx [hunk 1]
```diff
@@ -29,6 +29,13 @@ export function ChatListItem({
   const {currentAccount} = useSession()
   const menuControl = useMenuControl()
   const {gtMobile} = useBreakpoints()
+  const otherUser = convo.members.find(
+    member => member.did !== currentAccount?.did,
+  )
+  const isDeletedAccount = otherUser?.handle === 'missing.invalid'
+  const displayName = isDeletedAccount
+    ? 'Deleted Account'
+    : otherUser?.displayName ?? otherUser?.handle
```
Comment (mozzius) [inline src/screens/Messages/List/ChatListItem.tsx:38]: Should be
```js
  const displayName = isDeletedAccount
    ? 'Deleted Account'
    : otherUser?.displayName || otherUser?.handle
```

**Review by mozzius — CHANGES_REQUESTED** [review]
Minor display name logic problem

File: src/screens/Messages/List/ChatListItem.tsx (hunk 1 above)
Comment (mozzius) [reply src/screens/Messages/List/ChatListItem.tsx:38]: How do you get it to do the "accept suggested changes" button?

Comment (haileyok) [reply src/screens/Messages/List/ChatListItem.tsx:38]: Why bluesky why

**Review by mozzius — APPROVED** [review]
le sigh

**Pull request merged**